### ISSUE TITLE
Bable Node Setup Not Working

### ISSUE THREAD

**Issue by hillonyechekwa** [PacktPublishing/Full-Stack-Web-Development-with-GraphQL-and-React-Second-Edition#4]
I'm in chapter two where we setup a node server. I've installed bable-node and added it into the package.json just the way it was directed in the book but I keep getting this error.

this is how my package.json looks:
```
{
  "name": "graphbook",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "client": "webpack serve --devtool inline-source-map --hot --config webpack.client.config.js",
    "client:build": "webpack --config webpack.client.build.config.js",
    "server": "nodemon --exec babel-node --watch src/server src/server/index.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "express": "^4.18.1",
    "nodemon": "^2.0.20",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-helmet": "^6.1.0"
  },
  "devDependencies": {
    "@babel/core": "^7.19.1",
    "@babel/node": "^7.19.1",
    "@babel/preset-env": "^7.19.1",
    "@babel/preset-react": "^7.18.6",
    "babel-loader": "^8.2.5",
    "clean-webpack-plugin": "^4.0.0",
    "css-loader": "^6.7.1",
    "fileloader": "^2.0.0",
    "html-webpack-plugin": "^5.5.0",
    "mini-css-extract-plugin": "^2.6.1",
    "style-loader": "^3.3.1",
    "url-loader": "^4.1.1",
    "webpack": "^5.74.0",
    "webpack-cli": "^4.10.0",
    "webpack-dev-server": "^4.11.0"
  }
}
```
This is the result I get. 
```
 npm run server

> graphbook@1.0.0 server
> nodemon --exec babel-node --watch src/server src/server/index.js

[nodemon] 2.0.20
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): src\server\**\*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `babel-node src/server/index.js`
(node:3012) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
C:\Users\Hill\Documents\logicX\vestech\codacognito\reactql\qlseb\graphbook\src\server\index.js:1
import express from 'express';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Module._compile (C:\Users\Hill\Documents\logicX\vestech\codacognito\reactql\qlseb\graphbook\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.newLoader [as .js] (C:\Users\Hill\Documents\logicX\vestech\codacognito\reactql\qlseb\graphbook\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.runMain (node:internal/modules/run_main:81:12)
    at Object.<anonymous> (C:\Users\Hill\Documents\logicX\vestech\codacognito\reactql\qlseb\graphbook\node_modules\@babel\node\src\_babel-node.ts:215:12)
[nodemon] app crashed - waiting for file changes before starting...
```

I checked the npm docs for babel-node and there's no `--exec` option for babel node.  I would love I can get help with this issue.

**Comment by SeanPCarter** [comment]
Have you compared src\server\index.js to the source code?  It looks like the problem might be there.

**Comment by sebastiangrebe** [comment]
Can you try checking against the version from chapter 2? https://github.com/PacktPublishing/Full-Stack-Web-Development-with-GraphQL-and-React-Second-Edition/blob/main/Chapter02/package.json

Also maybe just do a diff of your files to the ones in this repo just to confirm

**Comment by SeanPCarter** [comment]
Thanks for the response, Sebastian,

I'll give it a try and let you know how I do.

spc
________________________________

**Comment by sebastiangrebe** [comment]
Closing this as there was no further update, feel free to create a new issue if you still notice it
